wfe/wfe.go

```diff
@@ -1421,7 +1421,7 @@ func (wfe *WebFrontEndImpl) Registration(ctx context.Context, logEvent *web.Requ
 	}
 }
 
-func (wfe *WebFrontEndImpl) deactivateAuthorization(ctx context.Context, authz *core.Authorization, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) bool {
+func (wfe *WebFrontEndImpl) deactivateAuthorization(ctx context.Context, authz *core.Authorization, authzPB *corepb.Authorization, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) bool {
```

Comment:
Hmm, not a fan of taking both the authz and the authzPB. How about only taking the authzPB? Line 1431 can get the RegistrationID from the protobuf just as easily, and the function can either modify the protobuf directly (line 1455, although it would be good to document that it performs that mutation in a top-level comment) or return a protobuf with the Status field changed.